### ISSUE TITLE
--coff not being applied when different values are provided

### ISSUE THREAD

**Issue by eaamgh** [Lolliedieb/lolMiner-releases#1887]
I've been trying to set different OC settings for different cards. An example:
   --coff 210,220,210 --cclk 2000,2100,2000  --mclk 800,810,810 --pl 150, 170, 150
  
Every param works as expected except for --coff.  lolminer will set cclk/mclk/pl to each card as specified. But -coff is only applied to the first card. No value is applied to any card but the first. It ignores the param for all other cards.

Setting a single value for --coff works:
--coff 210 --cclk 2000,2100,2000  --mclk 800,810,810 --pl 150, 170, 150

In this example, a 210 core offset is applied to all cards and the other params applied as expected.  
Am I missing something here ? Shouldn't -coff work like the other params ?

Thanks.

**Comment by eaamgh** [comment]
Tried with multiple versions, including 1.69

**Comment by jgonzis** [comment]
Which nvidia driver version?

Can you show the photo of the start?

**Comment by eaamgh** [comment]
Hi, I changed the rig since then. I was using 3 Nvidia cards. Moved one of them to another rig. This is something different, but still params not being applied. This time not with --coff but with --fan. Using HiveOS, driver 525.85.05 and lolminer 1.69: 

--devices NVIDIA --coff 210,245 --cclk 1485,2450 --mclk 810,2000 --pl 120,120 --fan 35,35

![image](https://user-images.githubusercontent.com/39251140/224384590-28307839-448e-4a29-9dab-c6f8f48eb54c.png)

This rig has 3 cards. First card (GPU0) is AMD, not included. Noticed that in this case --coff is applied correctly but --fan is only applied to GPU1. No message regarding fan for GPU2. Something similar was happening before but with --coff instead. And lolminer would not apply a number of settings to GPU1 and above. Hope it helps.
